feature/staticroute/otg_tests/static_route_test.go

```diff
@@ -66,41 +90,12 @@ func TestGNMI(t *testing.T) {
 	}
 
 	Ni1 := &telemetry.NetworkInstance{}
-
 	Ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").
 		GetOrCreateStatic("10.0.0.0/24").
 		GetOrCreateNextHop("h1").NextHop = telemetry.UnionString("192.0.2.23")
```

Comment:
If you adopt the approach to use `map[string]attrs.Attributes` from above then this could be made a bit clearer to the test reader, since you could have `...NextHop = telemetry.UnionString(atePorts["dst3"].IPv4)`.

Making this kind of change helps the person that is examining the test later not need to keep track of the IPs that are assigned themselves.